news/2024/2024-02-05-skin-of-the-year-2023-results.md

```diff
@@ -0,0 +1,84 @@
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/Divide-by-8.jpg)
+
+[星塵 DIVIDE ÷ 8](https://osu.ppy.sh/community/forums/topics/1792739), another entry from "Icons of an Era", expands on the theme presented by the background on [Sotarks' set of quaver](https://osu.ppy.sh/beatmapsets/873811#osu/1859185) by utilising a mix of very dark and bright colours. The beautiful tones of purple present throughout the background of most of the skin represent the darkness of the universe as we know it, with huge expanses of nothingness followed by beautiful, bright stars. For this stunning entry [thetasigma](https://osu.ppy.sh/users/6234482) earned themselves **767** votes.
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/Paper-Touhou.jpg)
+
+[Touhou Project ~ Paper Touhou](https://osu.ppy.sh/community/forums/topics/1746993) is a unique blend of the Touhou Project and the "paper" art-style that's been made popular by Nintendo ever since the year 2000. [CloudKeyz](https://osu.ppy.sh/users/15194624) used the original Paper Mario for the N64 as an inspiration and combined many of the more well known Touhou characters in order to craft a skin that has no right being as cute as it is, and for that community awarded them **782** votes.
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/Everlasting-Moment.jpg)
+
+Despite the fact they have only published 3 skins so far, [Chromasia](https://osu.ppy.sh/users/7306251) has only released highly polished, quality works of art. So it is no wonder that their entry for "Icons of an Era" has also been received **799** votes for this year's Skin of the Year contest. [Everlasting Moment](https://osu.ppy.sh/community/forums/topics/1792308) focuses on art used behind [Seni's iteration of *Chronostasis*](https://osu.ppy.sh/beatmapsets/1392153#osu/2874408) by FA [Release Hallucination](https://osu.ppy.sh/beatmaps/artists/194), captures the essence very well, and expands upon the theme set up by the background. Definitely be on the lookout for their future releases!
+
+![](/wiki/shared/news/2024-02-05-skin-of-the-year-2023-results/IT_S-SHOWTIME.jpg)
+
+You really shouldn't be surprised that thetasigma is once again on this list, as they are a mainstay in the skinning community with many contest winning entries under their belt.
+Their 2nd winning entry, [IT'S SHOWTIME! feat. Hatsune Miku](https://osu.ppy.sh/community/forums/topics/1864406), was released at the very end of 2023, but still managed to get **827** votes, which really shows how much their skins are beloved by the community. The skin itself has several art-styles sprinkled throughout the skin, ranging from graffiti art to 8 bit skin elements, and of course features the lovely Hatsune Miku! Surely they won't have another entry on this list... right?
```

Comment:
`Their 2nd winning entry` confused me for a moment as I thought this was related to his contest podium placements, which are more than 2, but this means 2nd within this contest. Not sure if its worth an edit

Reply:
i think its fine because its saying this is their 2nd entry that got top 10, and right under is the 3rd one. i wont resolve for now to get more opinions